gravitee-apim-e2e/lib/jest-utils.ts

```diff
@@ -125,4 +125,36 @@ export const describeIf = (condition) => (condition ? describe : describe.skip);
 export const describeIfV3 = describeIf(process.env.V4_EMULATION_ENGINE_DEFAULT == 'no');
 export const describeIfV4EmulationEngine = describeIf(process.env.V4_EMULATION_ENGINE_DEFAULT == 'yes');
 
+/**
+ * Check if debug tests should be skipped due to bridge compatibility.
+ * Debug tests are not compatible with client versions < 4.8.x when bridge server is >= 4.10.x
+ */
+function shouldSkipDebugTests(): boolean {
+  const clientTag = process.env.APIM_CLIENT_TAG;
+  if (!clientTag) {
+    return false;
+  }
+
+  // Extract version from tag (e.g., "4.7.x-latest", "graviteeio@4.7.0")
+  let versionStr = clientTag;
+  if (clientTag.includes('@')) {
+    versionStr = clientTag.split('@')[1];
+  }
+  versionStr = versionStr.replace('-latest', '');
+
+  // Parse major.minor version
+  const match = versionStr.match(/^(\d+)\.(\d+)/);
+  if (!match) {
+    return false;
+  }
```

Comment:
it means that running e2e tests on "master-latest" won't execute debug tests?
I think we should handle "master" apart.